user/enterprise/high-availability.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Enterprise High Availability Mode
+layout: en_enterprise
+
+---
+
+Travis CI Enterprise typically runs as a single container instance communicating with one or multiple workers and generally, this setup is quite stable and maintainable. However, in some cases -- especially volume licensing or by corporate requirement -- you might need to install Travis CI in High Availability (HA) mode. If you're interested, or might be interested, in HA, please let [sales@travis-ci.com](mailto:sales@travis-cicom?subject:HA%2FMode) sales know and we'll help you clarify and get started. 
+
+## Overview of Installation
+
+The platform installation is quite similar to the standard [Enterprise installation](/user/enterprise/installation#Setting-up-the-Travis-CI-Enterprise-Platform), and the [worker installation is identical](#Setting-up-the-Travis-CI-Enterprise-Worker). However, there are some additional [system prerequisites](/user/enterprise/prerequisites/#High-Availability-Mode), which mean that to install in HA mode, you will need the following:
+ * 3+ **16 gigs ofRAM and 8 CPUs**, i.e.`c3.2xlarge`'s - 2+ for the VMs running the Platform, and 1+ for the VMs running the Worker
```

Comment:
`c3.2xlarge` is not available anymore in most regions, therefore I am hesitant with recommending these here. In other doc pages we're recommending `c4.2xlarge` together with a 40GB HDD.

Reply:
@schultyy ah, good point -- I'll fix it here. Let's add an element to the mega issue to clean this up